cmd/connector/main.go

```diff
@@ -52,7 +53,11 @@ func main() {
 	defer client.Close()
 
 	// Watcher
-	watch := watcher.New(client, connectionsPrefix, locksPrefix, logger.Named("Watcher"))
+	watch := kv.NewWatcher(
+		namespace.NewKV(client, connectionsPrefix),
+		namespace.NewWatcher(client, connectionsPrefix),
+		namespace.NewKV(client, locksPrefix),
+		logger.Named("KV.Watcher"))
```

Comment:
I would put the last parenthesis on a newline, `)`

Reply:
I tend to not fix formatting if it's not enforced by lint.